app/models/user.rb

```diff
@@ -2,6 +2,6 @@ class User
   WRONG_PASSWORD = "Wrong password, sorry!"
 
   def bad
-    Rails.logger.warn("Hello")
+    Rails.logger.warn(user.email)
```

Comment:
🚫 **[Bearer]** <[ruby_rails_logger](https://docs.bearer.com/reference/rules/ruby_rails_logger)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>
# Sensitive data sent to Rails loggers detected.
## Description
Leaking sensitive data to loggers is a common cause of data leaks and can lead to data breaches. This rule looks for instances of sensitive data sent to rails loggers.

## Remediations
❌ Avoid using sensitive data in logger messages:

```ruby
Rails.logger.info('User is: #{user.email}')
```

✅ If you need to identify a user, ensure to use their unique identifier instead of their personal identifiable information:

```ruby
Rails.logger.info('User is: #{user.uuid}')
```

## Resources
- [OWASP logging cheat sheet](https://cheatsheetseries.owasp.org/cheatsheets/Logging_Cheat_Sheet.html)